diff -up Sphinx-1.2.3/sphinx/builders/linkcheck.py.old Sphinx-1.2.3/sphinx/builders/linkcheck.py
--- Sphinx-1.2.3/sphinx/builders/linkcheck.py.old	2015-10-13 15:15:40.767811124 +0200
+++ Sphinx-1.2.3/sphinx/builders/linkcheck.py	2015-10-13 15:14:41.916145677 +0200
@@ -17,7 +17,13 @@ import threading
 from os import path
 from urllib2 import build_opener, unquote, Request, \
     HTTPError, HTTPRedirectHandler
-from HTMLParser import HTMLParser, HTMLParseError
+from HTMLParser import HTMLParser
+
+try:
+    from HTMLParser import HTMLParseError
+except ImportError:
+    class HTMLParseError(Exception):
+        pass
 
 from docutils import nodes
 
