# HG changeset patch
# Parent 5da6d572bd088b04711b3bf70991c976d8f9c605

Index: Sphinx-0.6.5/setup.py
===================================================================
--- Sphinx-0.6.5.orig/setup.py
+++ Sphinx-0.6.5/setup.py
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
-import ez_setup
-ez_setup.use_setuptools()
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    import ez_setup
+    ez_setup.use_setuptools()
+    from setuptools import setup, find_packages
 
 import os
 import sys
-from setuptools import setup, find_packages
 from distutils import log
 
 import sphinx
